azurecompute/src/main/java/org/jclouds/azurecompute/util/ConflictManagementPredicate.java

```diff
@@ -178,7 +177,12 @@ public final boolean apply(final String input) {
          throw new RuntimeException(new TimeoutException(requestId));
       }
 
-      return operationSucceeded.apply(requestId);
+      try {
+         return operationSucceeded.apply(requestId);
+      } catch (RuntimeException e) {
+         logger.error("Operation '%s' failed with error '%s'", requestId, e.getMessage());
```

Comment:
I prefer to always include the cause - e.g. `logger.error(e, "Operation '%s' failed with error '%s'", requestId, e.getMessage());` (or whatever the ordering is in jclouds).

Imagine that the RuntimeException is a NullPointerException for some reason - our logger.error would say `failed with error ''`, and would not give any indication of the exception type or the line number it happened on.